code/modules/mob/living/carbon/human/human.dm

```diff
@@ -188,6 +188,11 @@
 		var/obj/item/organ/internal/stomach/stomach = internal_organs_by_name[BP_STOMACH]
 		if(stomach)
 			return stomach.ingested
+
+	if(should_have_organ(BP_REACTOR))
```

Comment:
Have you accounted for the chemical reactions that should/should not occur if IPCs ingest reagents?